api/envoy/api/v2/core/config_source.proto

```diff
@@ -65,6 +65,10 @@ message ApiConfigSource {
   // For GRPC APIs, the rate limit settings. If present, discovery requests made by Envoy will be
   // rate limited.
   RateLimitSettings rate_limit_settings = 6;
+
+  // Skip the node identifier in subsequent discovery requests for streaming gRPC config types.
+  // This will be enabled by default in the future.
```

Comment:
I'm generally not a fan (for stable API version reasons) of changing defaults in the API. I think we can deprecate this at some point and in v3 make it the default.

Reply:
Done